target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-xx8300.dtsi

```diff
@@ -0,0 +1,325 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/*
+ * Device Tree Source for Linksys xx8300 (Dallas)
+ *
+ * Copyright (C) 2019 Jeff Kletsky
+ * Updated 2020 Hans Geiblinger
+ *
+ */
+
+/dts-v1/;
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+	//
+	// OEM U-Boot provides either
+	// init=/sbin/init rootfstype=ubifs ubi.mtd=11,2048 \
+	//                 root=ubi0:ubifs rootwait rw
+	// or the same with ubi.mtd=13,2048
+	//
+
+/ {
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_0 rootfstype=squashfs ro";
+	};
+
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x10000000>;
```

Comment:
Is this needed?
U-boot should append this

Reply:
I can test a compilation without it, to see if we can remove it. I tried to leave as much of Jeff Kletsky's work in tact from his original DTS file (for the ea8300) when creating the DTSI for ea8300/mr8300. So I assume at the time of the original DTS file creation, he felt the need for it to be there.

Reply:
Please try, there is no need to leave stuff if not needed.

Reply:
Working on it now, stay tuned.....

Reply:
Compilation is clean without it, and system boots as expected. Kernel boot log is also clean.
![image](https://user-images.githubusercontent.com/22136734/92759648-f5d35f80-f35d-11ea-8ef7-68fcf1cdfed4.png)

`[    0.000000] Booting Linux on physical CPU 0x0
[    0.000000] Linux version 5.4.63 (openwrt@debian-openwrt) (gcc version 8.4.0 (OpenWrt GCC 8.4.0 r14443+2-b691e0b255)) #0 SMP Thu Sep 10 12:53:51 2020
[    0.000000] CPU: ARMv7 Processor [410fc075] revision 5 (ARMv7), cr=10c5387d
[    0.000000] CPU: div instructions available: patching division code
[    0.000000] CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
[    0.000000] OF: fdt: Machine model: Linksys MR8300 (Dallas)
[    0.000000] Memory policy: Data cache writealloc
[    0.000000] On node 0 totalpages: 130560
[    0.000000]   Normal zone: 1152 pages used for memmap
[    0.000000]   Normal zone: 0 pages reserved
[    0.000000]   Normal zone: 130560 pages, LIFO batch:31
[    0.000000] percpu: Embedded 15 pages/cpu s30092 r8192 d23156 u61440
[    0.000000] pcpu-alloc: s30092 r8192 d23156 u61440 alloc=15*4096
[    0.000000] pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 129408
[    0.000000] Kernel command line: init=/sbin/init rootfstype=ubifs ubi.mtd=11,2048 root=ubi0:ubifs rootwait rw root=/dev/ubiblock0_0 rootfstype=squashfs ro
[    0.000000] Dentry cache hash table entries: 65536 (order: 6, 262144 bytes, linear)
[    0.000000] Inode-cache hash table entries: 32768 (order: 5, 131072 bytes, linear)
[    0.000000] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.000000] Memory: 508276K/522240K available (5519K kernel code, 185K rwdata, 1456K rodata, 1024K init, 233K bss, 13964K reserved, 0K cma-reserved, 0K highmem)
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
[    0.000000] rcu: Hierarchical RCU implementation.
[    0.000000] rcu: RCU calculated value of scheduler-enlistment delay is 10 jiffies.
[    0.000000] NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
[    0.000000] random: get_random_bytes called from start_kernel+0x2f0/0x498 with crng_init=0
[    0.000000] arch_timer: cp15 timer(s) running at 48.00MHz (virt).
[    0.000000] clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0xb11fd3bfb, max_idle_ns: 440795203732 ns
[    0.000007] sched_clock: 56 bits at 48MHz, resolution 20ns, wraps every 4398046511096ns
[    0.000021] Switching to timer-based delay loop, resolution 20ns
[    0.000236] Calibrating delay loop (skipped), value calculated using timer frequency.. 96.00 BogoMIPS (lpj=480000)
[    0.000253] pid_max: default: 32768 minimum: 301
[    0.000449] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes, linear)
[    0.000468] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes, linear)
[    0.001497] CPU: Testing write buffer coherency: ok
[    0.002509] Setting up static identity map for 0x80300000 - 0x80300060
[    0.002647] rcu: Hierarchical SRCU implementation.
[    0.003089] smp: Bringing up secondary CPUs ...
[    0.006172] smp: Brought up 1 node, 4 CPUs
[    0.006191] SMP: Total of 4 processors activated (384.00 BogoMIPS).
[    0.006199] CPU: All CPU(s) started in SVC mode.
[    0.010938] VFP support v0.3: implementor 41 architecture 2 part 30 variant 7 rev 5
[    0.011094] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.011119] futex hash table entries: 1024 (order: 4, 65536 bytes, linear)
[    0.011384] pinctrl core: initialized pinctrl subsystem
[    0.012458] NET: Registered protocol family 16
[    0.012833] DMA: preallocated 256 KiB pool for atomic coherent allocations
[    0.014024] cpuidle: using governor ladder
[    0.014071] cpuidle: using governor menu
[    0.034617] cryptd: max_cpu_qlen set to 1000
[    0.037993] usbcore: registered new interface driver usbfs
[    0.038076] usbcore: registered new interface driver hub
[    0.038165] usbcore: registered new device driver usb
[    0.038216] pps_core: LinuxPPS API ver. 1 registered
[    0.038225] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.038251] PTP clock support registered
[    0.038940] workqueue: max_active 576 requested for napi_workq is out of range, clamping between 1 and 512
[    0.040129] clocksource: Switched to clocksource arch_sys_counter
[    0.040969] thermal_sys: Registered thermal governor 'step_wise'
[    0.041257] NET: Registered protocol family 2
[    0.041879] tcp_listen_portaddr_hash hash table entries: 512 (order: 0, 6144 bytes, linear)
[    0.041925] TCP established hash table entries: 4096 (order: 2, 16384 bytes, linear)
[    0.041987] TCP bind hash table entries: 4096 (order: 3, 32768 bytes, linear)
[    0.042075] TCP: Hash tables configured (established 4096 bind 4096)
[    0.042191] UDP hash table entries: 256 (order: 1, 8192 bytes, linear)
[    0.042228] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes, linear)
[    0.042480] NET: Registered protocol family 1
[    0.042519] PCI: CLS 0 bytes, default 64
[    0.044518] workingset: timestamp_bits=14 max_order=17 bucket_order=3
[    0.050018] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.050033] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
[    0.104516] qcom-pcie 40000000.pci: host bridge /soc/pci@40000000 ranges:
[    0.104565] qcom-pcie 40000000.pci:    IO 0x40200000..0x402fffff -> 0x40200000
[    0.104595] qcom-pcie 40000000.pci:   MEM 0x40300000..0x40ffffff -> 0x40300000
[    0.381703] qcom-pcie 40000000.pci: Link up
[    0.381858] qcom-pcie 40000000.pci: PCI host bridge to bus 0000:00
[    0.381880] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.381898] pci_bus 0000:00: root bus resource [io  0x0000-0xfffff] (bus address [0x40200000-0x402fffff])
[    0.381911] pci_bus 0000:00: root bus resource [mem 0x40300000-0x40ffffff]
[    0.381954] pci 0000:00:00.0: [17cb:1001] type 01 class 0x060400
[    0.381993] pci 0000:00:00.0: reg 0x10: [mem 0x00000000-0x00000fff]
[    0.382067] pci 0000:00:00.0: PME# supported from D0 D3hot
[    0.383626] PCI: bus0: Fast back to back transfers disabled
[    0.383867] pci 0000:01:00.0: [168c:0056] type 00 class 0x028000
[    0.384086] pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x001fffff 64bit]
[    0.384798] pci 0000:01:00.0: PME# supported from D0 D3hot
[    0.386494] PCI: bus1: Fast back to back transfers disabled
[    0.386544] pci 0000:00:00.0: BAR 8: assigned [mem 0x40400000-0x405fffff]
[    0.386561] pci 0000:00:00.0: BAR 0: assigned [mem 0x40300000-0x40300fff]
[    0.386582] pci 0000:01:00.0: BAR 0: assigned [mem 0x40400000-0x405fffff 64bit]
[    0.386659] pci 0000:00:00.0: PCI bridge to [bus 01-ff]
[    0.386675] pci 0000:00:00.0:   bridge window [mem 0x40400000-0x405fffff]
[    0.389429] pcieport 0000:00:00.0: AER: enabled with IRQ 94
[    0.391233] bam-dma-engine 8e04000.dma: num-channels unspecified in dt
[    0.391251] bam-dma-engine 8e04000.dma: num-ees unspecified in dt
[    0.392572] tcsr 1949000.tcsr: setting wifi_glb_cfg = 41000000
[    0.392663] tcsr 194b000.tcsr: setting usb hs phy mode select = e700e7
[    0.392744] tcsr 1953000.ess_tcsr: setting ess interface select = 0
[    0.392828] tcsr 1957000.tcsr: setting wifi_noc_memtype_m0_m2 = 2222222
[    0.393155] Serial: 8250/16550 driver, 2 ports, IRQ sharing disabled
[    0.393703] msm_serial 78af000.serial: msm_serial: detected port #0
[    0.393749] msm_serial 78af000.serial: uartclk = 1843200
[    0.393801] 78af000.serial: ttyMSM0 at MMIO 0x78af000 (irq = 24, base_baud = 115200) is a MSM
[    0.393828] msm_serial: console setup on port #0
[    1.022239] printk: console [ttyMSM0] enabled
[    1.027155] msm_serial: driver initialized
[    1.036030] loop: module loaded
[    1.037567] nand: device found, Manufacturer ID: 0xef, Chip ID: 0xda
[    1.038182] nand: Winbond W29N02GV
[    1.044811] nand: 256 MiB, SLC, erase size: 128 KiB, page size: 2048, OOB size: 64
[    1.048088] 16 fixed-partitions partitions found on MTD device qcom_nand.0
[    1.055568] Creating 16 MTD partitions on "qcom_nand.0":
[    1.062436] 0x000000000000-0x000000100000 : "sbl1"
[    1.069800] 0x000000100000-0x000000200000 : "mibib"
[    1.074377] 0x000000200000-0x000000300000 : "qsee"
[    1.079142] 0x000000300000-0x000000380000 : "cdt"
[    1.083518] 0x000000380000-0x000000400000 : "appsblenv"
[    1.088218] 0x000000400000-0x000000480000 : "ART"
[    1.093311] 0x000000480000-0x000000680000 : "appsbl"
[    1.099610] 0x000000680000-0x000000700000 : "u_env"
[    1.103208] 0x000000700000-0x000000740000 : "s_env"
[    1.107468] 0x000000740000-0x000000780000 : "devinfo"
[    1.112529] 0x000000780000-0x000005f80000 : "kernel"
[    1.125154] random: fast init done
[    1.202967] 0x000000a80000-0x000005f80000 : "rootfs"
[    1.286844] mtd: device 11 (rootfs) set to be root filesystem
[    1.287187] mtdsplit: no squashfs found in "rootfs"
[    1.291638] 0x000005f80000-0x00000b780000 : "alt_kernel"
[    1.382900] 0x000006280000-0x00000b780000 : "alt_rootfs"
[    1.466598] 0x00000b780000-0x00000b880000 : "sysdiag"
[    1.468477] 0x00000b880000-0x00000ff00000 : "syscfg"
[    1.541411] libphy: ipq40xx_mdio: probed
[    1.572720] ESS reset ok!
[    1.605699] ESS reset ok!
[    2.023724] libphy: Fixed MDIO Bus: probed
[    2.025437] EDMA using MAC@ - using
[    2.025450] 8e:2b:f6:a0:2c:4b
[    2.032800] i2c /dev entries driver
[    2.035169] sdhci: Secure Digital Host Controller Interface driver
[    2.036519] sdhci: Copyright(c) Pierre Ossman
[    2.042817] sdhci-pltfm: SDHCI platform and OF driver helper
[    2.050651] NET: Registered protocol family 10
[    2.054253] Segment Routing with IPv6
[    2.057258] NET: Registered protocol family 17
[    2.060998] 8021q: 802.1Q VLAN Support v1.8
[    2.065383] Registering SWP/SWPB emulation handler
[    2.078927] ubi0: attaching mtd11
[    2.175309] UBI: EOF marker found, PEBs from 113 will be erased
[    2.257861] ubi0: scanning is finished
[    2.285769] ubi0: volume 1 ("rootfs_data") re-sized from 9 to 525 LEBs
[    2.287440] ubi0: attached mtd11 (name "rootfs", size 85 MiB)
[    2.291341] ubi0: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[    2.297051] ubi0: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[    2.303924] ubi0: VID header offset: 2048 (aligned 2048), data offset: 4096
[    2.310743] ubi0: good PEBs: 680, bad PEBs: 0, corrupted PEBs: 0
[    2.317448] ubi0: user volume: 2, internal volumes: 1, max. volumes count: 128
[    2.323763] ubi0: max/mean erase counter: 1/0, WL threshold: 4096, image sequence number: 1599742431
[    2.330794] ubi0: available PEBs: 0, total reserved PEBs: 680, PEBs reserved for bad PEB handling: 40
[    2.340106] ubi0: background thread "ubi_bgt0d" started, PID 103
[    2.341416] block ubiblock0_0: created from ubi0:0(rootfs)
[    2.355272] hctosys: unable to open rtc device (rtc0)
[    2.372287] VFS: Mounted root (squashfs filesystem) readonly on device 254:0.
[    2.373948] Freeing unused kernel memory: 1024K
[    2.400588] Run /sbin/init as init process
[    3.082500] init: Console is alive
[    3.082737] init: - watchdog -
[    3.097140] random: crng init done
[    4.504721] kmodloader: loading kernel modules from /etc/modules-boot.d/*
[    4.558042] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    4.563616] SCSI subsystem initialized
[    4.566887] ehci-fsl: Freescale EHCI Host controller driver
[    4.568421] ehci-platform: EHCI generic platform driver
[    4.577901] dwc3-qcom 8af8800.usb3: IRQ hs_phy_irq not found
[    4.577939] dwc3-qcom 8af8800.usb3: IRQ dp_hs_phy_irq not found
[    4.583835] dwc3-qcom 8af8800.usb3: IRQ dm_hs_phy_irq not found
[    4.589428] dwc3-qcom 8af8800.usb3: IRQ ss_phy_irq not found
[    4.597093] dwc3-qcom 60f8800.usb2: IRQ hs_phy_irq not found
[    4.601267] dwc3-qcom 60f8800.usb2: IRQ dp_hs_phy_irq not found
[    4.606874] dwc3-qcom 60f8800.usb2: IRQ dm_hs_phy_irq not found
[    4.612548] dwc3-qcom 60f8800.usb2: IRQ ss_phy_irq not found
[    4.620610] dwc3 8a00000.dwc3: Failed to get clk 'ref': -2
[    4.787139] dwc3 6000000.dwc3: Failed to get clk 'ref': -2
[    5.141526] xhci-hcd xhci-hcd.0.auto: xHCI Host Controller
[    5.143215] xhci-hcd xhci-hcd.0.auto: new USB bus registered, assigned bus number 1
[    5.153326] xhci-hcd xhci-hcd.0.auto: hcc params 0x0228f665 hci version 0x100 quirks 0x0000000002010010
[    5.156266] xhci-hcd xhci-hcd.0.auto: irq 97, io mem 0x08a00000
[    5.199237] hub 1-0:1.0: USB hub found
[    5.201104] hub 1-0:1.0: 1 port detected
[    5.216676] xhci-hcd xhci-hcd.0.auto: xHCI Host Controller
[    5.218099] xhci-hcd xhci-hcd.0.auto: new USB bus registered, assigned bus number 2
[    5.223092] xhci-hcd xhci-hcd.0.auto: Host supports USB 3.0 SuperSpeed
[    5.235791] usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
[    5.263211] hub 2-0:1.0: USB hub found
[    5.265506] hub 2-0:1.0: 1 port detected
[    5.272139] xhci-hcd xhci-hcd.1.auto: xHCI Host Controller
[    5.272193] xhci-hcd xhci-hcd.1.auto: new USB bus registered, assigned bus number 3
[    5.276749] xhci-hcd xhci-hcd.1.auto: hcc params 0x0220f665 hci version 0x100 quirks 0x0000000002010010
[    5.284186] xhci-hcd xhci-hcd.1.auto: irq 98, io mem 0x06000000
[    5.294335] hub 3-0:1.0: USB hub found
[    5.299392] hub 3-0:1.0: 1 port detected
[    5.303532] xhci-hcd xhci-hcd.1.auto: xHCI Host Controller
[    5.307262] xhci-hcd xhci-hcd.1.auto: new USB bus registered, assigned bus number 4
[    5.312654] xhci-hcd xhci-hcd.1.auto: Host supports USB 3.0 SuperSpeed
[    5.320618] usb usb4: We don't know the algorithms for LPM for this host, disabling LPM.
[    5.328342] hub 4-0:1.0: USB hub found
[    5.335012] hub 4-0:1.0: config failed, hub doesn't have any ports! (err -19)
[    5.342753] usbcore: registered new interface driver usb-storage
[    5.345904] kmodloader: done loading kernel modules from /etc/modules-boot.d/*
[    5.352576] init: - preinit -
[    6.293801] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[   10.688216] mount_root: loading kmods from internal overlay
[   10.709777] kmodloader: loading kernel modules from //etc/modules-boot.d/*
[   10.711508] kmodloader: done loading kernel modules from //etc/modules-boot.d/*
[   11.593853] UBIFS (ubi0:1): default file-system created
[   11.594472] UBIFS (ubi0:1): Mounting in unauthenticated mode
[   11.598099] UBIFS (ubi0:1): background thread "ubifs_bgt0_1" started, PID 196
[   11.669302] UBIFS (ubi0:1): UBIFS: mounted UBI device 0, volume 1, name "rootfs_data"
[   11.669394] UBIFS (ubi0:1): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[   11.676327] UBIFS (ubi0:1): FS size: 65392640 bytes (62 MiB, 515 LEBs), journal size 3301376 bytes (3 MiB, 26 LEBs)
[   11.686179] UBIFS (ubi0:1): reserved for root: 3088656 bytes (3016 KiB)
[   11.696376] UBIFS (ubi0:1): media format: w5/r0 (latest is w5/r0), UUID 78AFD2C3-E1D9-4A07-893F-539B162E515D, small LPT model
[   11.703931] block: attempting to load /tmp/ubifs_cfg/upper/etc/config/fstab
[   11.714525] block: unable to load configuration (fstab: Entry not found)
[   11.721221] block: attempting to load /tmp/ubifs_cfg/etc/config/fstab
[   11.728147] block: unable to load configuration (fstab: Entry not found)
[   11.734525] block: attempting to load /etc/config/fstab
[   11.744615] block: unable to load configuration (fstab: Entry not found)
[   11.746160] block: no usable configuration
[   11.756069] UBIFS (ubi0:1): un-mount UBI device 0
[   11.756978] UBIFS (ubi0:1): background thread "ubifs_bgt0_1" stops
[   11.765501] UBIFS (ubi0:1): Mounting in unauthenticated mode
[   11.768004] UBIFS (ubi0:1): background thread "ubifs_bgt0_1" started, PID 197
[   11.843290] UBIFS (ubi0:1): UBIFS: mounted UBI device 0, volume 1, name "rootfs_data"
[   11.843380] UBIFS (ubi0:1): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[   11.850241] UBIFS (ubi0:1): FS size: 65392640 bytes (62 MiB, 515 LEBs), journal size 3301376 bytes (3 MiB, 26 LEBs)
[   11.860052] UBIFS (ubi0:1): reserved for root: 3088656 bytes (3016 KiB)
[   11.870353] UBIFS (ubi0:1): media format: w5/r0 (latest is w5/r0), UUID 78AFD2C3-E1D9-4A07-893F-539B162E515D, small LPT model
[   12.650626] block: attempting to load /tmp/ubifs_cfg/upper/etc/config/fstab
[   12.650814] block: unable to load configuration (fstab: Entry not found)
[   12.656490] block: attempting to load /tmp/ubifs_cfg/etc/config/fstab
[   12.663515] block: unable to load configuration (fstab: Entry not found)
[   12.669745] block: attempting to load /etc/config/fstab
[   12.676573] block: unable to load configuration (fstab: Entry not found)
[   12.681525] block: no usable configuration
[   12.690058] mount_root: overlay filesystem has not been fully initialized yet
[   12.692732] mount_root: switching to ubifs overlay
[   12.728522] overlayfs: upper fs does not support xattr, falling back to index=off and metacopy=off.
[   12.734227] urandom-seed: Seed file not found (/etc/urandom.seed)
[   12.809750] procd: - early -
[   12.809864] procd: - watchdog -
[   13.440655] procd: - watchdog -
[   13.441036] procd: - ubus -
[   13.602253] procd: - init -
[   14.089915] kmodloader: loading kernel modules from /etc/modules.d/*
[   14.096918] ntfs: driver 2.1.32 [Flags: R/O MODULE].
[   14.105956] urngd: v1.0.2 started.
[   14.150763] Mirror/redirect action on
[   14.159569] u32 classifier
[   14.159596]     input device check on
[   14.161221]     Actions configured
[   14.173990] Loading modules backported from Linux version v5.8-0-gbcf876870b95
[   14.174028] Backport generated by backports.git v5.8-1-0-g79400d9e
[   14.216669] usbcore: registered new interface driver ums-alauda
[   14.217878] usbcore: registered new interface driver ums-cypress
[   14.234374] usbcore: registered new interface driver ums-datafab
[   14.238868] usbcore: registered new interface driver ums-freecom
[   14.244327] usbcore: registered new interface driver ums-isd200
[   14.250422] usbcore: registered new interface driver ums-jumpshot
[   14.255539] usbcore: registered new interface driver ums-karma
[   14.263224] usbcore: registered new interface driver ums-sddr09
[   14.268095] usbcore: registered new interface driver ums-sddr55
[   14.274286] usbcore: registered new interface driver ums-usbat
[   14.285124] wireguard: WireGuard 1.0.20200908 loaded. See www.wireguard.com for information.
[   14.285194] wireguard: Copyright (C) 2015-2019 Jason A. Donenfeld <Jason@zx2c4.com>. All Rights Reserved.
[   14.341852] xt_time: kernel timezone is -0000
[   14.419931] PPP generic driver version 2.4.2
[   14.421293] NET: Registered protocol family 24
[   14.448985] ath10k 5.4 driver, optimized for CT firmware, probing pci device: 0x56.
[   14.449113] ath10k_mac_create, priv_size: 868  hw: d67de258  hw->priv: bd201c16
[   14.456107] ath10k_pci 0000:01:00.0: enabling device (0140 -> 0142)
[   14.463498] ath10k_pci 0000:01:00.0: pci irq msi oper_irq_mode 2 irq_mode 0 reset_mode 0
[   15.935200] ath10k_pci 0000:01:00.0: qca9888 hw2.0 target 0x01000000 chip_id 0x00000000 sub 0000:0000
[   15.935247] ath10k_pci 0000:01:00.0: kconfig debug 0 debugfs 1 tracing 0 dfs 1 testmode 0
[   15.947347] ath10k_pci 0000:01:00.0: firmware ver 10.4b-ct-9888-fW-13-8c5b2baa2 api 5 features mfp,peer-flow-ctrl,txstatus-noack,wmi-10.x-CT,ratemask-CT,regdump-CT,txrate-CT,flush-all-CT,pingpong-CT,ch-regs-CT,nop-CT,set-special-CT,tx-rc-CT,cust-stats-CT,txrate2-CT,beacon-cb-CT,wmi-block-ack-CT,wmi-bcn-rc-CT crc32 cbf49903
[   16.264306] ath10k_pci 0000:01:00.0: board_file api 2 bmi_id 0:23 crc32 b050e9d6
[   18.300379] ath10k_pci 0000:01:00.0: unsupported HTC service id: 1536
[   18.301364] ath10k_pci 0000:01:00.0: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   18.305822] ath10k_pci 0000:01:00.0: msdu-desc: 2500  skid: 32
[   18.361681] ath10k_pci 0000:01:00.0: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   18.362779] ath10k_pci 0000:01:00.0: wmi print 'free: 114572 iram: 12804 sram: 29508'
[   18.614156] ath10k_pci 0000:01:00.0: htt-ver 2.2 wmi-op 6 htt-op 4 cal pre-cal-file max-sta 32 raw 0 hwcrypto 1
[   18.750766] ath: EEPROM regdomain: 0x834b
[   18.750775] ath: EEPROM indicates we should expect a country code
[   18.750786] ath: doing EEPROM country->regdmn map search
[   18.750792] ath: country maps to regdmn code: 0x3a
[   18.750797] ath: Country alpha2 being used: US
[   18.750801] ath: Regpair used: 0x3a
[   18.756635] ath10k_mac_create, priv_size: 924  hw: fe82ed3a  hw->priv: 980ac37a
[   20.525472] ath10k_ahb a000000.wifi: qca4019 hw1.0 target 0x01000000 chip_id 0x003b00ff sub 0000:0000
[   20.525519] ath10k_ahb a000000.wifi: kconfig debug 0 debugfs 1 tracing 0 dfs 1 testmode 0
[   20.537031] ath10k_ahb a000000.wifi: firmware ver 10.4b-ct-4019-fW-13-8c5b2baa2 api 5 features mfp,peer-flow-ctrl,txstatus-noack,wmi-10.x-CT,ratemask-CT,regdump-CT,txrate-CT,flush-all-CT,pingpong-CT,ch-regs-CT,nop-CT,set-special-CT,tx-rc-CT,cust-stats-CT,txrate2-CT,beacon-cb-CT,wmi-block-ack-CT,wmi-bcn-rc-CT crc32 1f3ed50f
[   20.612201] ath10k_ahb a000000.wifi: board_file api 2 bmi_id 0:20 crc32 8fda7e22
[   21.918917] ath10k_ahb a000000.wifi: unsupported HTC service id: 1536
[   21.919877] ath10k_ahb a000000.wifi: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   21.924388] ath10k_ahb a000000.wifi: msdu-desc: 2500  skid: 32
[   21.973335] ath10k_ahb a000000.wifi: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   21.976622] ath10k_ahb a000000.wifi: wmi print 'free: 53252 iram: 13576 sram: 35752'
[   22.119033] ath10k_ahb a000000.wifi: htt-ver 2.2 wmi-op 6 htt-op 4 cal pre-cal-file max-sta 32 raw 0 hwcrypto 1
[   22.289572] ath: EEPROM regdomain: 0x834b
[   22.289580] ath: EEPROM indicates we should expect a country code
[   22.289590] ath: doing EEPROM country->regdmn map search
[   22.289597] ath: country maps to regdmn code: 0x3a
[   22.289602] ath: Country alpha2 being used: US
[   22.289606] ath: Regpair used: 0x3a
[   22.293471] ath10k_mac_create, priv_size: 924  hw: f3ef9ae2  hw->priv: e2b25420
[   23.075852] ath10k_ahb a800000.wifi: qca4019 hw1.0 target 0x01000000 chip_id 0x003b00ff sub 0000:0000
[   23.075903] ath10k_ahb a800000.wifi: kconfig debug 0 debugfs 1 tracing 0 dfs 1 testmode 0
[   23.087804] ath10k_ahb a800000.wifi: firmware ver 10.4b-ct-4019-fW-13-8c5b2baa2 api 5 features mfp,peer-flow-ctrl,txstatus-noack,wmi-10.x-CT,ratemask-CT,regdump-CT,txrate-CT,flush-all-CT,pingpong-CT,ch-regs-CT,nop-CT,set-special-CT,tx-rc-CT,cust-stats-CT,txrate2-CT,beacon-cb-CT,wmi-block-ack-CT,wmi-bcn-rc-CT crc32 1f3ed50f
[   23.140201] ath10k_ahb a800000.wifi: board_file api 2 bmi_id 0:21 crc32 8fda7e22
[   24.447462] ath10k_ahb a800000.wifi: unsupported HTC service id: 1536
[   24.448907] ath10k_ahb a800000.wifi: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   24.452981] ath10k_ahb a800000.wifi: msdu-desc: 2500  skid: 32
[   24.501103] ath10k_ahb a800000.wifi: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   24.502103] ath10k_ahb a800000.wifi: wmi print 'free: 53252 iram: 13576 sram: 35752'
[   24.640432] ath10k_ahb a800000.wifi: htt-ver 2.2 wmi-op 6 htt-op 4 cal pre-cal-file max-sta 32 raw 0 hwcrypto 1
[   24.800918] ath: EEPROM regdomain: 0x834b
[   24.800926] ath: EEPROM indicates we should expect a country code
[   24.800935] ath: doing EEPROM country->regdmn map search
[   24.800944] ath: country maps to regdmn code: 0x3a
[   24.800949] ath: Country alpha2 being used: US
[   24.800952] ath: Regpair used: 0x3a
[   24.813293] kmodloader: done loading kernel modules from /etc/modules.d/*
[   59.785342] br-lan: port 1(eth0) entered blocking state
[   59.785387] br-lan: port 1(eth0) entered disabled state
[   59.789871] device eth0 entered promiscuous mode
[   59.814464] ess_edma c080000.edma: eth1: GMAC Link is down
[   60.790288] br-lan: port 1(eth0) entered blocking state
[   60.790357] br-lan: port 1(eth0) entered forwarding state
[   60.796143] IPv6: ADDRCONF(NETDEV_CHANGE): br-lan: link becomes ready`